src/OpenLoco/Economy.cpp

```diff
@@ -1,21 +1,90 @@
 #include "Economy.h"
 #include "Interop/Interop.hpp"
+#include "Ui/WindowManager.h"
+#include "Ui/WindowType.h"
 
 using namespace OpenLoco::Interop;
 
 namespace OpenLoco::Economy
 {
+    // Inflation rates?
+    static const uint32_t _4FDEC0[32] = {
```

Comment:
Shall we name this array then?
```suggestion
    static const uint32_t _inflationFactors[32] = {
```